### ISSUE TITLE
Chromium always takes 34s to start

### ISSUE THREAD

**Issue by jeffrson** [linuxmint/linuxmint#564]
Chromium version: 109.0.5414.119~linuxmint1+vera

I upgraded from LM 21 to 21.1, DE is Cinnamon.

Then I installed chromium via software manager (which had not been installed previously) from main mint repository, not flathub.

Now I realize that every start of chromium takes pretty much 34 seconds - always. Chromium is showing a title bar with a window replicating a part of desktop background (wallpaper).

Adding a new window to a running instance happens instantly (duration of a window animation). It's definitely not only the first time after booting, but every time!

When I start firefox, it only takes a second to start the whole application.

Hardware is DELL precision 7550 - on demand switching from Intel to Nvidia. It keeps running on Intel afaict.

PS: I had another issue when I started chromium "too early" after login: it did not start at all, frozen at the window with title bar and wallpaper cutout. Only the obvious renderer process could be closed - a lot of /usr/lib/chromium/chrome_crashpad_handler and /usr/lib/chromium/chromium stayed running, the latter even "unkillable" (had to reboot). Don't know if this is related though...

PPS: tried Google Chrome from https://www.google.com/intl/en_uk/chrome/ - it behaves the same, so the problem is probably not with Chromium but with the system itself. AFAICT, there's nothing special with it...

PPPS: Hmmm, now I found it seems to work with `chromium --disable-gpu --disable-software-rasterizer` or `google-chrome -disable-gpu --disable-software-rasterizer`, respectively. However, I don't know, which conclusions I should draw from it. What does it mean for my system (faulty hardware config?) and is there a workaround (other than always appending those arguments)?

**Comment by mtwebster** [comment]
Maybe make a new user and see if it occurs there. Also, running from a terminal might give some clues if there are warnings.

**Comment by jeffrson** [comment]
No, there are no warnings at all - neither in terminal nor in .xsession-errors, unfortunately.

I have yet to try with a new user, but I can't imagine how this should change anything.

**Comment by mtwebster** [comment]
Maybe not, but it's a simple test that will rule out anything in your user directory being a factor.

**Comment by jeffrson** [comment]
Okay - so, no, it didn't help, unfortunately.

However, more research revealed that `prime-select intel` as well as removing nvidia driver (currently nvidia-driver-525-open) also kind of "fixes" this problem. It now takes "only" 2 seconds to start a new instance of chromium or chrome (note <1s for firefox).

I wonder if this is a driver bug, and if so, what to do about it.

**Comment by mtwebster** [comment]
What kernel are you using?
